tests/playwright/sanity/modules/container/container-1.test.ts

```diff
@@ -82,7 +82,7 @@ test.describe( 'Container tests @container', () => {
 		expect.soft( elBeforeButton ).toEqual( elAfterHeading );
 	} );
 
-	test.skip( 'Test widgets display inside the container using various directions and content width', async ( { page, apiRequests }, testInfo ) => {
+	test( 'Test widgets display inside the container using various directions and content width', async ( { page, apiRequests }, testInfo ) => {
```

Comment:
@Svitlana-Dykun Do you have any idea what caused the differences between the screenshots?

How long have these tests been skipped for?